packages/@eventual/aws-runtime/src/utils.ts

```diff
@@ -283,8 +283,12 @@ export function entityServiceTableSuffix(entityName: string) {
   return `entity-${entityName}`;
 }
 
-export function bucketServiceBucketSuffix(bucketName: string) {
-  return `bucket-${bucketName}`;
+export function bucketServiceBucketSuffix(
+  bucketName: string,
+  accountID: string,
+  region: string
+) {
+  return `bucket-${bucketName}-${accountID}-${region}`;
```

Comment:
i think the `bucket-` prefix is totally unnecessary.

Reply:
Updated